graylog2-server/src/main/java/org/graylog/security/shares/EntityShareResponse.java

```diff
@@ -98,6 +103,9 @@ public static Builder create() {
         @JsonProperty("missing_permissions_on_dependencies")
         public abstract Builder missingPermissionsOnDependencies(Map<GRN, Collection<EntityDescriptor>> missingDependencies);
 
+        @JsonProperty("synced_dependent_entities")
+        public abstract Builder syncedDependentEntities(Set<GRN> dependentEntities);
```

Comment:
Let's remove the dependency terminology to avoid confusion.

```suggestion
        @JsonProperty("synced_entities")
        public abstract Builder syncedEntities(Set<GRN> dependentEntities);
```